integrations/ad_integration/ad_sync.py

```diff
@@ -29,37 +30,20 @@
 
 
 class AdMoSync(object):
-    def __init__(self):
+    def __init__(self, all_settings=None):
         logger.info('AD Sync Started')
-        cfg_file = pathlib.Path.cwd() / 'settings' / 'settings.json'
-        if not cfg_file.is_file():
-            raise Exception('No setting file')
-        self.settings = json.loads(cfg_file.read_text())
+
+        self.settings = all_settings
+        if self.settings is None:
+            self.settings = read_ad_conf_settings.read_settings()
+
```

Comment:
This is gonna be an issue

Reply:
Så må vi jo have set på det. Der er også meget med multiple_ad, branchen, der skal kværnes igennem.

Reply:
Jeg tror dette her bør være ækvivalent med hvad der var tidligere, udover at det nu er lazy